File: Java/algoritmos/Mdc.java
public class Mdc {
    public static int calcularMdc(int numero, int numero2) {
        int maior = Math.max(numero, numero2);
        int menor = Math.min(numero, numero2);

        while (menor != 0) {
            int resto = maior % menor;
            maior = menor;
            menor = resto;
        }

        return maior;
    }
}

File: JS/Mdc.js
const prompt = require('prompt-sync')();

function MDC(){
    const numero = parseInt(prompt("Digite o primeiro número: "));
    const numero2 = parseInt(prompt("Digite o segundo número: "));

    let maior = numero;
    let menor = numero2;

    if(numero < numero2){
        maior = numero2;
        menor = numero;
    }

    while (menor !== 0) {
        let resto = maior % menor;
        maior = menor;
        menor = resto;
    }
    
    return maior;
}


console.log(MDC());